src/shutdown.c

```diff
@@ -0,0 +1,84 @@
+  /**
+   *    ||          ____  _ __
+   * +------+      / __ )(_) /_______________ _____  ___
+   * | 0xBC |     / __  / / __/ ___/ ___/ __ `/_  / / _ \
+   * +------+    / /_/ / / /_/ /__/ /  / /_/ / / /_/  __/
+   *  ||  ||    /_____/_/\__/\___/_/   \__,_/ /___/\___/
+   *
+   * Crazyflie 2.1 NRF Firmware
+   * Copyright (c) 2021, Bitcraze AB, All rights reserved.
+   *
+   * This library is free software; you can redistribute it and/or
+   * modify it under the terms of the GNU Lesser General Public
+   * License as published by the Free Software Foundation; either
+   * version 3.0 of the License, or (at your option) any later version.
+   *
+   * This library is distributed in the hope that it will be useful,
+   * but WITHOUT ANY WARRANTY; without even the implied warranty of
+   * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   * Lesser General Public License for more details.
+   *
+   * You should have received a copy of the GNU Lesser General Public
+   * License along with this library.
+   */
+
+#include "syslink.h"
+
+#include "pm.h"
+#include "systick.h"
+
+#define TIMEOUT_TICKS 1000 // 100 ms
```

Comment:
this might be too long, anyone got a feel? Could this be 50ms? 25ms? @ataffanel ?

Reply:
Switching OFF is mainly a human-driven operation and a human cannot really make the difference between 100ms and 50ms. So I would give more chance to the SD-card to flush and keep it at 100ms.